src/components/ChartTitle.jsx

```diff
@@ -13,44 +13,54 @@ const ChartTitle = ({
     isPriceUp,
     currentSymbol,
     Menu,
-    CategoricalDisplay,
+    MarketSelector,
     AnimatedPrice,
     onCloseMenu,
-    isMobile,
```

Comment:
`isMobile` is retrieved from the chart store in `MenuStore`, so this prop is redundant.